src/lib/dialog/dialog.md

```diff
@@ -1,7 +1,7 @@
 The `MdDialog` service can be used to open modal dialogs with Material Design styling and 
 animations.
 
-<!-- example(dialog-overview) -->
+<!-- example(dialog-basic) -->
```

Comment:
Is this supposed to be `dialog-overview`?

I think we should probably get rid of this "dialog-basic" example because it is _too_ basic; it doesn't really show anything. The first example really needs to showcase the component.